src/net/sourceforge/kolmafia/FamiliarData.java

```diff
@@ -783,6 +847,10 @@ public String getRace() {
     return this.race;
   }
 
+  public boolean getActive() {
```

Comment:
Naming conventions elsewhere would have this as `isActive` as it's a boolean, e.g. https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter#5322666

I don't know what we do in the rest of the codebase, so this is just a thought.

Reply:
I agree.